eZ/Bundle/EzPublishRestBundle/Resources/config/routing.yml

```diff
@@ -95,6 +95,15 @@ ezpublish_rest_copyContent:
     requirements:
         contentId: \d+
 
+ezpublish_rest_deleteContentTranslation:
+    path: /content/objects/{contentId}/{languageCode}
+    defaults:
+        _controller: ezpublish_rest.controller.content:deleteContentTranslation
+    methods: [DELETE]
+    requirements:
+        contentId: \d+
+        languageCode: '[a-z]{3}-[A-Z]{2}'
```

Comment:
As long as it's not been implemented it should be ok. However try to get hold of @bdunogier to check if he has any objections.